Sources/Valkey/ValkeyClient.swift

```diff
@@ -148,19 +153,20 @@ extension ValkeyClient {
         isolation: isolated (any Actor)? = #isolation,
         operation: (ValkeyConnection) async throws -> sending Value
     ) async throws -> Value {
-        let connection = try await self.leaseConnection()
-
-        defer { self.connectionPool.releaseConnection(connection) }
-
-        return try await operation(connection)
+        fatalError()
```

Comment:
And how are you planning to support withConnection now that the connection request is `ValkeyConnectionRequest<RESPToken>`